src/System.Security.Claims/src/System/Security/Claims/Claim.cs

```diff
@@ -31,15 +30,15 @@ private enum SerializationMask
         }
 
         [NonSerialized]
```

Comment:
nit: prefer whitespace (extra line) before and after field if it contains attribute:
```
field1

[myattribute]
field2

field3
```